reviews/serializers.py

```diff
@@ -41,6 +41,32 @@ def __repr__(self):
     return IfCanViewLogCreatorField
 
 
+class ReviewableCountingRelationshipField(RelationshipField):
```

Comment:
Are there conventions for DRF serializer names?

Reply:
Not that I know of, save ending them with `Field`. I'm open to suggestions

Reply:
Having a verb in the name seems odd to me -- maybe `ReviewableCountsRelationshipField` (if it's actually weird).

Reply:
It's very Java -esque, which is a bit odd in python I think `ReviewableCountsRelationshipField` makes more sense as a field name.